pkg/connector/connector.go

```diff
@@ -29,7 +29,10 @@ import (
 
 const githubDotCom = "https://github.com"
 
-var ValidAssetDomains = []string{"avatars.githubusercontent.com"}
+var (
+	ValidAssetDomains     = []string{"avatars.githubusercontent.com"}
+	maxPageSize       int = 100 // maximum page size github supported.
+)
```

Comment:
Wow. Apparently go-github has no way to set a default page size, so people either manually set it everywhere or use a library: https://github.com/gofri/go-github-pagination